examples/bind/game.amp.html

```diff
@@ -0,0 +1,206 @@
+              (G.grid[0][0] == 0 && G.grid[1][0] > 0 ? G.grid[1][0] :
+                  (G.grid[0][0] > 0 && G.grid[1][0] == 0 ? G.grid[0][0] :
+                      G.grid[0][0]
+                  )
+              )
+          ),
+          (G.grid[0][1] > 0 && G.grid[0][1] == G.grid[1][1] ? G.grid[0][1] + G.grid[1][1] :
+              (G.grid[0][1] == 0 && G.grid[1][1] > 0 ? G.grid[1][1] :
+                  (G.grid[0][1] > 0 && G.grid[1][1] == 0 ? G.grid[0][1] :
+                      G.grid[0][1]
+                  )
+              )
+          )
+        ],
+        [
+          (G.grid[0][0] > 0 && G.grid[0][0] == G.grid[1][0] ? (Math.random() < G.spawnChance ? 2 : 0) :
+              (G.grid[0][0] == 0 && G.grid[1][0] > 0 ? (Math.random() < G.spawnChance ? 2 : 0) :
+                  (G.grid[0][0] > 0 && G.grid[1][0] == 0 ? 0 :
+                      G.grid[1][0]
+                  )
+              )
+          ),
+          (G.grid[0][1] > 0 && G.grid[0][1] == G.grid[1][1] ? (Math.random() < G.spawnChance ? 2 : 0) :
+              (G.grid[0][1] == 0 && G.grid[1][1] > 0 ? (Math.random() < G.spawnChance ? 2 : 0) :
+                  (G.grid[0][1] > 0 && G.grid[1][1] == 0 ? 0 :
+                      G.grid[1][1]
+                  )
+              )
+          )
+        ]
+      ],
+      'spawnChance': G.spawnChance
+    }
+  })">UP</button>
+
+  <button on="tap:AMP.setState({
+    'G': {
+      'grid': [
+        [
+          (G.grid[0][0] > 0 && G.grid[0][0] == G.grid[1][0] ? (Math.random() < G.spawnChance ? 2 : 0):
+              (G.grid[0][0] == 0 && G.grid[1][0] > 0 ? (Math.random() < G.spawnChance ? 2 : 0) :
+                  (G.grid[0][0] > 0 && G.grid[1][0] == 0 ? 0 :
+                      G.grid[0][0]
+                  )
+              )
+          ),
+          (G.grid[0][1] > 0 && G.grid[0][1] == G.grid[1][1] ? (Math.random() < G.spawnChance ? 2 : 0) :
+              (G.grid[0][1] == 0 && G.grid[1][1] > 0 ? (Math.random() < G.spawnChance ? 2 : 0) :
+                  (G.grid[0][1] > 0 && G.grid[1][1] == 0 ? 0 :
+                      G.grid[0][1]
+                  )
+              )
+          )
+        ],
+        [
+          (G.grid[0][0] > 0 && G.grid[0][0] == G.grid[1][0] ? G.grid[0][0] + G.grid[1][0] :
+              (G.grid[0][0] == 0 && G.grid[1][0] > 0 ? G.grid[1][0] :
+                  (G.grid[0][0] > 0 && G.grid[1][0] == 0 ? G.grid[0][0] :
+                      G.grid[1][0]
+                  )
+              )
+          ),
+          (G.grid[0][1] > 0 && G.grid[0][1] == G.grid[1][1] ? G.grid[0][1] + G.grid[1][1] :
+              (G.grid[0][1] == 0 && G.grid[1][1] > 0 ? G.grid[1][1]  :
+                  (G.grid[0][1] > 0 && G.grid[1][1] == 0 ? G.grid[0][1] :
+                      G.grid[1][1]
+                  )
+              )
+          )
+        ]
+      ],
+      'spawnChance': G.spawnChance
+    }
+  })">DOWN</button>
```

Comment:
How would you feel about arranging the buttons into a natural UP, RIGHT, DOWN, LEFT shape instead of having them in a line?